src/Algebra/Graph/Export/Dot.hs

```diff
@@ -113,25 +118,28 @@ export :: (IsString s, Monoid s, Ord a, ToGraph g, ToVertex g ~ a) => Style a s
 export Style {..} g = render $ header <> body <> "}\n"
   where
     header    = "digraph" <+> literal graphName <> "\n{\n"
-    with x as = if null as then mempty else line (x <+> attributes as)
+    with x as = if null as then mempty else line (x <+> attributes quoting as)
     line s    = indent 2 s <> "\n"
     body      = unlines (map literal preamble)
              <> ("graph" `with` graphAttributes)
              <> ("node"  `with` defaultVertexAttributes)
              <> ("edge"  `with` defaultEdgeAttributes)
              <> E.export vDoc eDoc g
     label     = doubleQuotes . literal . vertexName
-    vDoc x    = line $ label x <+>                      attributes (vertexAttributes x)
-    eDoc x y  = line $ label x <> " -> " <> label y <+> attributes (edgeAttributes x y)
+    vDoc x    = line $ label x <+>                      attributes quoting (vertexAttributes x)
+    eDoc x y  = line $ label x <> " -> " <> label y <+> attributes quoting (edgeAttributes x y)
 
 -- | A list of attributes formatted as a DOT document.
 -- Example: @attributes ["label" := "A label", "shape" := "box"]@
```

Comment:
The example here should be adjusted. There may be other cases where the comments got out of date.